services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/subscription/SubscriptionManagerImpl.java

```diff
@@ -118,4 +124,34 @@ public Etag getEtag(String objectId, ObjectType objectType) {
 		etag.setEtag(changeDao.getEtag(objectIdLong, objectType));
 		return etag;
 	}
+
+	@Override
+	public SubscriberPagedResults getSubscribers(UserInfo userInfo, Topic topic, String nextPageToken) {
+		ValidateArgument.required(userInfo, "userInfo");
+		ValidateArgument.required(topic, "topic");
+		ValidateArgument.required(topic.getObjectId(), "Topic.objectId");
+		ValidateArgument.required(topic.getObjectType(), "Topic.objectType");
+		AuthorizationManagerUtil.checkAuthorizationAndThrowException(
+				authorizationManager.canSubscribe(userInfo, topic.getObjectId(), topic.getObjectType()));
+		NextPageToken token = null;
```

Comment:
add new constructor: NextPageToken(string token, long defaultLimit, long maxLimit)
update: NextPageToken(string token) does not throw exception on null token. Instead, call the above constructor with default values.
Add constants: DEFAULT_LIMIT (50), DEFAULT_OFFSET (0), MAX_LIMIT (50).